e2e/deploy-rook_test.go

```diff
@@ -0,0 +1,59 @@
+package e2e
```

Comment:
this need to be removed we are not doing unit testing for packages under E2E folder

Reply:
done!